### DISCUSSION TITLE
2023/08/13/QQMarkUse/

### DISCUSSION THREAD

**Discussion by giscus** [wu1015/wu1015.github.io#5] [Announcements]
# 2023/08/13/QQMarkUse/

本身特点 免费 可以在浏览器访问 支持文字+图片的形式进行写作，图片可以获取直链  作为图床  作为图床使用是最经典的使用，我自己博客图床用的就是QQ收藏。  具体使用方式：  创建收藏并加入需要的图片 分享给自己，进入链接后点击右上角复制链接（如果要从浏览器访问的话需要这一步） 浏览器打开链接，右键图片复制图片链接  当作一篇文章分享  同样经典的使用。优势是速度快，便于传播、编辑，而且不会像G

https://wu1015.github.io/2023/08/13/QQMarkUse/

**Comment by wu1015** [comment]
结束了。24.12.19开始，部分图片无法加载。腾讯收紧了口袋。
